plot_continuous_monitor.py

```diff
@@ -77,16 +78,17 @@ def plot_continuous_monitor(filename, open_graphs=False):
     running_compute_monotasks = 0
     if "Running Compute Monotasks" in json_data:
       running_compute_monotasks = json_data["Running Compute Monotasks"]
+    xvdf_running_disk_monotasks = 0
+    xvdb_running_disk_monotasks = 0
+    if "Running Disk Monotasks" in json_data:
```

Comment:
I am trying to parse logs from Spark, but Spark's ContinuousMonitor does not log "Running Disk Monotasks", so I added the above check to make sure that plot_continuous_monitor.py does not throw a key error. The code that I added is of the same form as the code for extracting "Running Compute Monotasks", "Running Macrotasks", etc.